cogs/utils/dataIO.py

```diff
@@ -9,6 +9,15 @@ class InvalidFileIO(Exception):
 class DataIO():
     def __init__(self):
         self.logger = logging.getLogger("red")
+        try:
+            __import__("aiofiles")
+            import aiofiles
```

Comment:
__import__ wont actually import, it just checks for the existance of the imported package.